Traceback (most recent call last):
  File "/Users/lucialarraona/Desktop/projects_sem2/dtudeeplearning22/02_Models/checking_dataset.py", line 311, in <module>
    train_loss, train_acc = train_model(model, train_iterator, epoch)
  File "/Users/lucialarraona/Desktop/projects_sem2/dtudeeplearning22/02_Models/checking_dataset.py", line 219, in train_model
    model.to('cuda')
  File "/Users/lucialarraona/opt/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 673, in to
    return self._apply(convert)
  File "/Users/lucialarraona/opt/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 387, in _apply
    module._apply(fn)
  File "/Users/lucialarraona/opt/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 409, in _apply
    param_applied = fn(param)
  File "/Users/lucialarraona/opt/anaconda3/lib/python3.8/site-packages/torch/nn/modules/module.py", line 671, in convert
    return t.to(device, dtype if t.is_floating_point() or t.is_complex() else None, non_blocking)
  File "/Users/lucialarraona/opt/anaconda3/lib/python3.8/site-packages/torch/cuda/__init__.py", line 164, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
